multimodal/tests/unittests/test_zero_shot.py

```diff
@@ -98,3 +103,36 @@ def test_clip_zero_shot():
     # invalid API usage 2: predicting probability with only one dictionary as input.
     with pytest.raises(AssertionError):
         prob = predictor.predict_proba({"image": [cat_image_name], "text": [cat_text]})
+
+
+def test_timm_zero_shot():
+    cat_image_name, dog_image_name = download_sample_images()
+
+    predictor = MultiModalPredictor(
+        hyperparameters={
```

Comment:
Can we test some CNN models as well? (maybe at least a ResNet50) CNN has some structural differences than transformers, that may break current code logic.

Reply:
Thanks. Added CNN tests.